src/core/namespace.c

```diff
@@ -74,7 +74,17 @@ typedef struct TargetMount {
 static const TargetMount protect_kernel_tunables_table[] = {
         { "/proc/sys",                  READONLY,       false, },
         { "/proc/sysrq-trigger",        READONLY,       true, /* Not always compiled into kernel */ },
+        { "/proc/latency_stats",        READONLY,       true, },
+        { "/proc/mtrr",                 READONLY,       true, },
+        { "/proc/apm",                  READONLY,       true, },
+        { "/proc/acpi",                 READONLY,       true, },
+        { "/proc/timer_stats",          READONLY,       true, },
+        { "/proc/asound",               READONLY,       true, },
+        { "/proc/bus",                  READONLY,       true, },
+        { "/proc/fs",                   READONLY,       true, },
+        { "/proc/irq",                  READONLY,       true, },
         { "/sys",                       READONLY,       false, },
+        { "/sys/kernel/debug",          READONLY,       true, },
```

Comment:
I think we should protect the `/sys/kernel/tracing` too.

https://lwn.net/Articles/630526/
> Finally, a new directory is created when tracefs is enabled called
/sys/kernel/tracing. This will be the new location that system admins
may mount tracefs if they are not using debugfs.

https://lkml.org/lkml/2016/5/13/327
> On my system, simply enabling and disabling function graph tracer can
crash the kernel

:-)